### DISCUSSION TITLE
Difficulty Rating

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1385] [Feature Request]
**Before submitting this feature request I have**
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Score recipes on how difficult they are to cook, using scale 1-5

**(Optional): Suggest A Solution**
Currently there are recipe ratings, a similar ranking system could be built along-side ratings to score difficulty via 1-5 stars. Currently I've solved it via tags titled "Difficulty x/5", though a built-in system would be better.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by epheterson** [comment]
In hindsight, using Easy/Medium/Hard tags are better than "Difficulty x/5", so if we implement it only those three levels would be necessary (and easier to read).

**Comment by Gecko-with-a-hat** [comment]
This would be an awesome feature for Mealie. Additionally it would be awesome if we could then sort or filter based on difficulty. This is great for my communities that are comprised of cooks of all levels.

**Comment by ChipSkylark37** [comment]
I use a tasklist app (TODY) that also has a similar rating using 'grime' - 1, 2, or 3. 1 is easy, 2 is medium and 3 is hard. 

Difficulty rating would be a nice addition, and then allow this to be filtered (I just want an easy dinner tonight, etc).
